### PR TITLE
Third party details on contact page

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

**Comment by axemonkey** [comment]
FE LGTM :+1:

File: cla_public/apps/contact/constants.py [hunk 1]
```diff
@@ -10,3 +10,13 @@
     (DAY_TODAY, _('Call me today at')),
     (DAY_SPECIFIC, _('Call me in the next week on'))
 )
+
+# TODO: GET this from cla_common
```
Comment (r4vi) [inline cla_public/apps/contact/constants.py]: maybe we can do this?

Comment (tyom) [inline cla_public/apps/contact/constants.py]: Ah, yes. I almost forgot.

Comment (tyom) [inline cla_public/apps/contact/constants.py]: Updated.

**Comment by r4vi** [comment]
Looks good :+1:

File: cla_public/apps/contact/forms.py [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+from flask import current_app
 from flask.ext.babel import lazy_gettext as _
 from flask_wtf import Form
 import pytz
 from wtforms import Form as NoCsrfForm
-from wtforms import BooleanField, FormField, RadioField, SelectField, \
+from wtforms import BooleanField, RadioField, SelectField, \
     StringField, TextAreaField
-from wtforms.validators import InputRequired, Optional, Length
+from wtforms.validators import InputRequired, Optional, Required, Length
```
Comment (andyhd) [inline cla_public/apps/contact/forms.py]: That's not alphabetical order >:/

**Comment by andyhd** [comment]
:shipit:

**Pull request merged**